Dockerfile

```diff
@@ -1,7 +1,7 @@
 # builder image
-FROM golang:1.12.4
+FROM golang:1.12.9
```

Comment:
Setting this to tag `1.12` will ensure we always take the latest without having to continually update the patch number